CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeService JokeService;
+        private IJokeOutput JokeOutServ;
+
+        public IJokeOutput JokeOut { get => JokeOutServ; set => JokeOutServ = value; }
+        public IJokeService JokeServ { get => JokeService; set => JokeService = value; }
```

Comment:
Because these properties don't add any logic in their getters or setters they could simply be auto properties and remove the need for fields.
```suggestion
        public IJokeOutput JokeOut { get; set; }
        public IJokeService JokeServ { get; set; }
```